js/utils/Logger/Logger.ts

```diff
@@ -2,36 +2,28 @@ module hirespace {
     'use strict';
 
     export class Logger {
-        static log(data: any) {
-            if (hirespace.Debug.getEnvironment() !== 'development') {
-                return false;
-            }
+        static execute(type: string, data: any, notify?: boolean) {
```

Comment:
Actually you're right, I am not testing this one component anymore, just the 4 below so will change to private :))